lib/skeleton/config/scripts/ios/build_app.rb

```diff
@@ -44,7 +44,7 @@
 
 # Choosing the SDK for device or simulator
 sdk = ''
-if ARGV[1] == 'device'
+if (ARGV[1] == 'devices' || ARGV[1] == 'device') 
```

Comment:
I see no point in this.
Moreover the help text shows 'device' and you didn't updated it, so no one will know about 'devices' without opening the file.